### PR TITLE
Added RadiusNeighborsClassifier

### PR DESCRIPTION
**Pull request by aadityasinha-dotcom**
### Issue Id you have worked upon - 
#48 

### CHANGES MADE -
Added  RadiusNeighborsClassifier.

### Made changes to -
"https://github.com/blobcity/autoai/blob/main/blobcity/config/classifier_config.py"

### NOTE -
Please consider this PR as a submission towards Hacktoberfest 2021 and add the hacktoberfest-accepted label to it.

### CODE REVIEW

**Comment by Thilakraj1998** [comment]
radius neighbor Classifier doesnt have n_neighbors has a parameter:

do refer offical documentation for parameters: https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.RadiusNeighborsClassifier.html

**Comment by aadityasinha-dotcom** [comment]
> radius neighbor Classifier doesnt have n_neighbors has a parameter:
> 
> do refer offical documentation for parameters: https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.RadiusNeighborsClassifier.html

Okay, I will recheck that.

**Comment by aadityasinha-dotcom** [comment]
@Thilakraj1998 I have removed n_neighbor. Is there any changes left that I have missed?

**Comment by Thilakraj1998** [comment]
@aadityasinha-dotcom 

it still missing some of the parameter configuration like radius, metric and outlier_label

**Comment by aadityasinha-dotcom** [comment]
@Thilakraj1998 It was only missing the radius parameter, I have added that. Is there anything else?

**Comment by Thilakraj1998** [comment]
@aadityasinha-dotcom  for metric parameter there are more then 1 distance metric 

do consider consider adding following distance metrics mentioned in the official documentation:
https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.DistanceMetric.html#sklearn.neighbors.DistanceMetric

**Comment by aadityasinha-dotcom** [comment]
> @aadityasinha-dotcom for metric parameter there are more then 1 distance metric
> 
> do consider consider adding following distance metrics mentioned in the official documentation: https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.DistanceMetric.html#sklearn.neighbors.DistanceMetric

I have added euclidean distance parameter in the metric.
